lib/miq_automation_engine/service_models/miq_ae_service_authentication_rhsm.rb

```diff
@@ -0,0 +1,6 @@
+module MiqAeMethodService
+  class MiqAeServiceAuthenticationRhsm < MiqAeServiceAuthentication
+    expose :password_encrypted
+    expose :userid
```

Comment:
@alongoldboim Database columns and virtual columns are automatically exposed through the service models, you do not need an `expose` statement for it.